editor/src/components/editor/import-utils.ts

```diff
@@ -366,3 +366,13 @@ function removeImportDetails(
     importedFromWithin: importedFromWithin,
   }
 }
+
+export function getProjectImports(projectContents: ProjectContentTreeRoot): {
```

Comment:
Looks like you never ended up using this function

Reply:
Removed in 112d15b20.